src/util/aws-util.ts

```diff
@@ -227,6 +227,9 @@ export class AwsUtil {
     }
 
     public static GetDefaultRegion(profileName?: string): string {
+        const regionFromEnv = process.env.AWS_REGION;
```

Comment:
Based on the [AWS CLI docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html), we should first check `AWS_DEFAUL_REGION` then `AWS_REGION`

Reply:
great! happy to add that. I reason i added `AWS_REGION` was mostly to use the current region in the context of a codebuild. i think `AWS_DEFAULT_REGION` > `AWS_REGION` might work well here too.